lib/fs.js

```diff
@@ -167,7 +167,7 @@ function nullCheck(path, callback) {
 }
 
 function isFd(path) {
-  return (path >>> 0) === path;
+  return (typeof path === 'number' && ~~path === path);
```

Comment:
Is this change faster? I'm not a fan of the old code but doing `~~path === path` should at least be motivated IMO.